source/material_model/visco_plastic.cc

```diff
@@ -1013,15 +1013,15 @@ namespace aspect
                                    "exponent and $T$ is temperature. "
                                    "This form of the viscosity equation is commonly used in "
                                    "geodynamic simulations. See, for example, Billen and Hirth "
-                                   "(2007), G3, 8, Q08012. Significantly, other studies may use"
-                                   "slightly different forms of the viscosity equation leading to"
-                                   "variations in how specific terms are defined or combined. For"
-                                   "example, the grain size exponent should always be positive in"
+                                   "(2007), G3, 8, Q08012. Significantly, other studies may use "
+                                   "slightly different forms of the viscosity equation leading to "
+                                   "variations in how specific terms are defined or combined. For "
+                                   "example, the grain size exponent should always be positive in "
                                    "the diffusion viscosity equation used here, while other studies"
```

Comment:
There's one space missing here at the end of this line, otherwise this is ready to go. 
Thank you, @naliboff!